### ISSUE TITLE
Make tooltip accessible

### ISSUE THREAD

**Issue by rachelruderman** [cypress-io/cypress-design#418]
Tooltip should always be accessible by keyboard users and should have option to be hidden for screenreader users

**Comment by rockhold** [comment]
The corresponding Jira logged for the A11y team that would benefit from an accessible tooltip component is at https://cypress-io.atlassian.net/browse/CP-2597. This ticket is not planned for Q2, but it's also not something we'd likely touch without the tooltip component being ready first.
